CHANGELOG.md

```diff
@@ -2,6 +2,84 @@
 
 About the past changelog entries, see [old CHANGELOG](CHANGELOG-v4.md) instead.
 
+## Release v5.2.0 - 2024/12/14
+
+### News
+
+* Update bundled Ruby to 3.2.6
+* Update bundled Fluentd to v1.18.0
+* Update bundled gems
+* deb rpm: Fixed to not execute v4 restart migration process unexpectedly.
+* msi: set GEM_HOME/GEM_PATH in fluentd.bat
+* Support upgrade fluentd service with zero downtime.
+  Note that you can use this feature when upgrade to the next version of fluent-package.
+
```

Comment:
How about adding the following 2 points?

```suggestion
* Support upgrade fluentd service with zero downtime.
  Note that you can use this feature when upgrade to the next version of fluent-package.
* fluentd.service: Remove GEM_HOME/GEM_PATH env vars because they are unnecessary. 
* deb: suppress service restart by needrestart.
  The package places `/etc/needrestart/conf.d/50-fluent-package.conf`.
```

Reply:
Good catch!